processing/src/main/java/io/druid/segment/NilColumnValueSelector.java

```diff
@@ -28,9 +29,11 @@
  * Represents "absent" column.
  */
 @PublicApi
-public final class NilColumnValueSelector implements ColumnValueSelector
+public class NilColumnValueSelector implements ColumnValueSelector
```

Comment:
Javadoc of getDouble/Long/Float is outdated.

Reply:
done.